hellsemble/hellsemble.py

```diff
@@ -210,6 +213,13 @@ def __fit_estimators_sequential(
                 X_fit[failed_observations_mask],
                 y_fit[failed_observations_mask],
             )
+
+            # Validate the ensemble
+            val_score = self.evaluate_hellsemble(X_val, y_val)
+            print(f'Validation score for sequential emsemble: {val_score}')
+            if val_score >= 0.95:
```

Comment:
it would be best if we make tihs `0.95` as an optional parameter in `fit`

Reply:
For the future: one fixed threshold for the metric across different datasets is quite risky. Depending on the metric and prior on y 0.95 can mean both good model and bad model